ddtrace/ext/test_visibility/item_ids.py

```diff
@@ -0,0 +1,40 @@
+"""Provides identifier classes for items used in the Test Visibility API
+
+NOTE: BETA - this API is currently in development and is subject to change.
```

Comment:
Maybe also include this in a release note as a new feature?

Reply:
I'm working on a PR for documentation, and then we'll put a release note.